login.php

```diff
@@ -0,0 +1,67 @@
+<?php
+// cek session
+session_start();
+if (isset($_SESSION["login"])) {
+    header("Location: home.php");
+    exit;
+}
+
+require 'functions.php';
+    
+    if (isset($_POST["login"])) {
+        $username = $_POST["username"];
+        $password = $_POST["password"];
+
+        $result = mysqli_query($conn, "SELECT * FROM user_account
+                             WHERE username = '$username'");
+    
+        // check username
+        if ( mysqli_num_rows($result) === 1 ) {
+            $row = mysqli_fetch_assoc($result);
+            if (password_verify($password, $row["password"])) {
+                // set session
+                $_SESSION["login"] = true;
+                
+                header("Location: index.php");
```

Comment:
udah berhasil login pake 
```
user : admin
pass : admin
```
tapi pas udah berhasil login, dia rediectnya ke index.php, filenya gada yak ? 404 not found errornya
![image](https://user-images.githubusercontent.com/10482274/179923190-221ea865-62bb-4b7a-830e-683adc57f611.png)